### DISCUSSION TITLE
加一个bt下载器

### DISCUSSION THREAD

**Discussion by YOUXICHUNMI** [Meloong-Git/PCL#1762] [提问] [answered]
在启动器内增加一个bt下载器

**Comment by Big-Cake-jpg** [comment, marked as answer]
PCL2 的主业是 Minecraft 启动器，而非下载器。

提供自定义下载功能只是一个副业，如有磁力链接或 BT 种子下载等需求，请使用 Aria2 或者迅雷等软件，而非 PCL2。

**Comment by huhujieshuo** [comment]
PCL2被各位硬生生整成了PCL2高速下载器（我看过一个留言，推荐一个高速下载器：PCL2[doge])

**Reply by SqrtTan** [reply]
真·Plain Craft Downloader

**Comment by xianhongtao** [comment]
需要下载磁力链接就使用qbittorrent吧
